aries-test-harness/features/present-proof-v3.feature

```diff
@@ -0,0 +1,33 @@
+@PresentProofV3 @DIDComm-V2 @UsesCustomParameters
+Feature: RFC 0454 Aries agent present proof v3
+
+   @T001-PresentProofV3 @RFC0510 @WACI @CredFormat_JSON-LD @Schema_Citizenship_Context @CredProposalStart
+   Scenario Outline: Present Proof of specific types and proof is acknowledged with a Citizenship credential type with a DID Exchange Connection
+      Given "2" agents
+         | name  | role     |
+         | Faber | verifier |
+         | Bob   | prover   |
+      And "Acme" is running with parameters "{"mime-type":["didcomm/v2"]}"
+      And "Bob" is running with parameters "{"mime-type":["didcomm/v2"]}"
+      And "Faber" is running with parameters "{"mime-type":["didcomm/v2"]}"
+      And "Acme" has issued a credential with <credential_data> to "Bob" using issue-credential v3
```

Comment:
I think `issuer credential v3` is not adding anything to the step declaration here. We already know it will be V3 based on the tags, most importantly the `@DIDcomm-V2` tag. We could capture that tag in code and use the proper issue_cred v3. The same pattern we did for Issue Credential v1 and v2 should be followed IMO. Thoughts?

Reply:
Don't we have different steps for issue-credential v1 and v2 already? I was continuing the pattern of having separate steps for separate protocols. I do agree though that it would be cleaner for the features if these were combined, and if the step implementation used a tag to determine behaviour.

Reply:
That is a valid point, but they were separated because in v2 we went on to implement the input data into the step. Issue Credential V1 tests don't have that. I can image if we went back to refactor those tests to include the data tables and examples to pass data in, we would probably reuse the code in issue credential v2.

Reply:
Just finished merging the v2/v3 steps, using tag logic as you suggested :)